src/components/ButtonDropdown/ButtonDropdown.tsx

```diff
@@ -168,11 +168,17 @@ export const ButtonDropdown = ({
     position === 'bottom-right' && styles['button-dropdown--bottom-right'],
     className,
```

Comment:
So it's being applied to both elements now?

Reply:
Good catch 🤦  will remove here